src/utils/Api.js

```diff
@@ -4,6 +4,6 @@ const ncApi = axios.create({
   baseURL: `https://nc-news-9ihg.onrender.com/api`,
 });
 
-export const fetchTopics = () => {
-  return ncApi.get(`/topics`);
+export const fetchArticles = () => {
```

Comment:
Great that you've separated out this function into it's own file, and great use of an axios instance 🙌 
This filename however shouldn't be capitalised as that implies it's a component 🔍

Reply:
Done, thanks